packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -533,8 +586,48 @@ class TimeInput extends React.Component {
     if ((event.key === '=' || event.key === '+') && !this.props.atMaxDate) {
       const currentTimeValue = this.formatHour(hour, meridiem).concat(':', minute).concat(this.props.showSeconds ? ':'.concat(second) : '');
       if (TimeUtil.validateTime(currentTimeValue, this.props.showSeconds)) {
-        if (minute === '59') {
-          minute = '00';
+        if (inputType === 3 && this.props.showSeconds) {
```

Comment:
what is this value 3 signifies ?

Reply:
3 signifies second (https://github.com/cerner/terra-framework/blob/main/packages/terra-time-input/src/TimeUtil.js#L421)